HSRPizzaHelper/View/SettingView.swift

```diff
@@ -68,7 +68,7 @@ struct SettingView: View {
                         }
                     }
 
-                    var url: String {
+                    var url: String = {
```

Comment:
Why editing this? Besides it should be an `let` instead of `var` if written like this.

Reply:
To make it compilable with Xcode 14.1.
I'll make it into a constant.